### PR TITLE
Demo threading visuals 2

### PR DESCRIPTION
**Pull request by ahayzen-kdab**

### CODE REVIEW

**Comment by ahayzen-kdab** [comment]
Check with fonts we actually need and any typos etc.

File: examples/demo_threading/images/RLogo.png.license [hunk 1]
```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```
Comment (LeonMatthesKDAB) [inline examples/demo_threading/images/RLogo.png.license:1]: We need to add a link to this license:
https://creativecommons.org/licenses/by/4.0/

See: https://foundation.rust-lang.org/policies/logo-policy-and-media-guide/

Same in the RLogoLarge license file

File: examples/demo_threading/images/kdabLogo.png.license [hunk 2]
```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Nuno Pinheiro <nuno@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
```
Comment (LeonMatthesKDAB) [inline examples/demo_threading/images/kdabLogo.png.license:4]: Stating MIT or Apache-2.0 here might be an issue, this is the KDAB logo, after all :thinking: 
We do *not* want to hand this out freely.

Comment (LeonMatthesKDAB) [reply examples/demo_threading/images/kdabLogo.png.license:4]: Maybe we should make this a new repository that we take in as a submodule a closed license :thinking:

**Review by LeonMatthesKDAB — CHANGES_REQUESTED** [review]
The code for this looks good, but I'm still concerned about the CopyRight of the Rust logo, as well as KDAB logos.

The Rust Logo basically just needs an update to the license file to link to the attribution license.

The KDAB logo is more problematic, as we do not want to put that under MIT/Apache2 license.
We could just mark its license as "closed source" or something, but I wonder whether SPDX would then mark our entire project as closed source :thinking: 
The libraries are definitely all open source and the upload to crates.io would only contain MIT/Apache2 licensed code.
However, SPDX may still be confused.

File: examples/demo_threading/images/kdabLogo.png.license (hunk 2 above)
Comment (ahayzen-kdab) [reply examples/demo_threading/images/kdabLogo.png.license:4]: Note that you can create custom licenses in the SPDX format, i've done this in other repositories

File: examples/demo_threading/images/RLogo.png.license (hunk 1 above)
Comment (ahayzen-kdab) [reply examples/demo_threading/images/RLogo.png.license]: Changed to `CC-BY-4.0`

File: examples/demo_threading/images/kdabLogo.png.license (hunk 2 above)
Comment (ahayzen-kdab) [reply examples/demo_threading/images/kdabLogo.png.license:4]: Need to figure out the license of this one :sweat_smile:

Comment (ahayzen-kdab) [reply examples/demo_threading/images/kdabLogo.png.license]: Done

**Review by LeonMatthesKDAB — APPROVED** [review]
LGTM

**Pull request merged**